idb/postgres/internal/writer/writer.go

```diff
@@ -309,12 +342,12 @@ func (w *Writer) AddBlock0(block *bookkeeping.Block) error {
 		_, err := results.Exec()
 		if err != nil {
 			results.Close()
-			return fmt.Errorf("AddBlock() exec err: %w", err)
+			return fmt.Errorf("AddBlock() exec err for block 0: %w", err)
```

Comment:
To ease grepping, and make it clear what function this is actually in:
```suggestion
			return fmt.Errorf("AddBlock0() exec err: %w", err)
```